addons/pinniped/config-controller/hack/ytt/schema.yaml

```diff
@@ -0,0 +1,4 @@
+#@data/values-schema
```

Comment:
i freakin love that we have a schema, so glad you added this. does it work tho? like, i would expect to see actual types below for `tkr` and `infrastructure_provider` as opposed to `""`

Reply:
I was thinking `""` still meant "a string" but let me check, I bet you are correct and this is insufficient.

Reply:
I think your point is correct! I thought `""` would still be "empty string" and work, but it didn't.  Updated, and now:

```
ytt -f ./hack/ytt -v tkr=TANZU.KAT.RHINOS -v iaas=IN.ANOTHER.AARDVARK.SNEEZE
ytt: Error: Overlaying data values (in following order: values.yaml, additional data values):
One or more data values were invalid
====================================

Given data value is not declared in schema
(data-value arg):
    |
  1 | iaas=IN.ANOTHER.AARDVARK.SNEEZE
    |

    = found: iaas
    = expected: one of { infrastructure_provider, tkr } (from schema.yaml:2)

```
Does indeed error if the value is not an expected value.